components/event_dispatcher.rst

```diff
@@ -348,9 +348,17 @@ of the event to dispatch::
     // creates the OrderPlacedEvent and dispatches it
     $event = new OrderPlacedEvent($order);
     $dispatcher->dispatch($event, OrderPlacedEvent::NAME);
+    // or since Symfony 4.3+
+    $dispatcher->dispatch($event);
+
+.. versionadded:: 4.3
+
+    The event name is now optional since Symfony 4.3 in
+    the :method:`Symfony\\Component\\EventDispatcher\\EventDispatcher::dispatch`.
```

Comment:
```suggestion
    The event name in :method:`Symfony\\Component\\EventDispatcher\\EventDispatcher::dispatch`
    is optional since Symfony 4.3.
```